cmd/secret/add.go

```diff
@@ -84,13 +84,8 @@ var AddCmd = cli.Command{
 		},
 		&cli.StringSliceFlag{
 			Name:    "event",
-			Usage:   "Secret limited to these events",
+			Usage:   "Secret limited to these events (by default push, tag, and deploy are enabled)",
```

Comment:
I know this is extremely small.. but, could we update `deploy` to `deployment` since that's the actual name of the event?

Out of curiosity, what are your thoughts on just leaving the default events populated?

I'm not opposed to this change 👍 But thinking we don't have to worry about populating/updating the Usage if we leave the default events.

Reply:
well, it's possible that there would be a mismatch between server defaults and defaults defined here, I think. That's partially why I don't like enumerating the events in case in the Usage.

Reply:
Sure 👍 that's fair!

Reply:
Might be overkill but we could make the defaults a `const` in types and then they would reference the same values. Just an idea though